revscoring/languages/persian.py

```diff
@@ -10,11 +10,43 @@
     raise ImportError("No enchant-compatible dictionary found for 'fa'.  " +
                       "Consider installing 'myspell-fa'.")
 
+REVERENTWORDS = set([
+    ur"آله", ur"فرموده?",ur"فرمودند",ur"السلام",ur"حضرت\b",ur"\([سعص]\)",ur"\(ره\)",ur"\(قس\)",
```

Comment:
change all ur"" to r"". Revscoring is only python3-based and python 3 throws syntax error in case of ur""